ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_onprem.py

```diff
@@ -224,9 +224,6 @@ class WCAOnPremRoleExplanationPipeline(
     def __init__(self, config: WCAOnPremConfiguration):
         super().__init__(config=config)
 
-    def invoke(self, params: RoleExplanationParameters) -> RoleExplanationResponse:
```

Comment:
This overrode the super class's implementation and was not needed.

Reply:
👍